lib/spack/docs/tables/system_prerequisites.csv

```diff
@@ -3,6 +3,7 @@ Python, 2.7/3.6-3.10, , Interpreter for Spack
 C/C++ Compilers, , , Building software
 make, , , Build software
 patch, , , Build software
+binutils, , , Build software
```

Comment:
I think this can be left out, since it's a dependency of gcc on linux and above it mentions C/C++ compilers. Or it should be called C/C++ compiler toolchain or so.

Reply:
Fair enough, I'm not very fussy about this part. I've removed it.